src/estimation/include/iDynTree/Estimation/BerdyHelper.h

```diff
@@ -128,7 +156,13 @@ enum BerdySensorTypes
     /**
      * Non-physical sensor that measures the wrench trasmitted by a joint.
      */
-    JOINT_WRENCH_SENSOR     = 1003
+    JOINT_WRENCH_SENSOR     = 1003,
+
+    /**
+     * Non-physical sensor that holds the value of Rate of Change of Momentum (ROCM) of the system
```

Comment:
Minor, for acronyms you typically capitalize all the letters that are in the acronym
```suggestion
     * Non-physical sensor that holds the value of Rate Of Change of Momentum (ROCM) of the system
```